### ISSUE TITLE
silent hill 3 PAL 60Hz, issue with opening cutscene

### ISSUE THREAD

**Issue by hcgamer** [PCSX2/pcsx2#112]
1) Did the game ever work correctly
as fa as I know, no

2) What steps will reproduce the problem?
important step is to select 60Hz after launching game, the issue only happens in 60Hz mode, then wait for the intro cutscene to play, with the guitar music

3) What exactly happens when you experience this issue (listing any console errors or screen output you receive)?
in hardware more, you have a black screen during intro
in software more, you have blinking screen during intro

4) What version of PCSX2 are you using? On what operating system? Plugins?
Windows 7 x64
PCSX2 1.2.1
GDSx

5) Please provide any additional information below
tested on a fresh install, all settings to default
tested with all speed hacks deactivated, no effect
my gfx card is Sapphire HD7870 with catalyst drivers v14.4
the intro plays fine however if you select 50Hz after launching game

**Comment by gregory38** [comment]
Issue is memory wrapping of the GS. It must be fine on the SW renderer (with a recent build). 
Of course on the HW renderer it is more complex to handle.

**Comment by FlatOutPS2** [comment]
Yes, Software mode is fine at 50 and 60 Hz. Hardware mode can be fixed for 50 Hz with Unscale Point&Line hack, Hardware mode is still a black screen at 60 Hz.

**Comment by JordanTheToaster** [comment]
Does this still happen on latest nightly?

**Comment by refractionpcsx2** [comment]
Still an issue and probably will be for a  good while. The game wraps around the end of GS memory, something that the TC isn't really equipped for.

The game currently has the SW FMV switch to compensate for the moment.

**Comment by bigol83** [comment]
As of pcsx2 2.1.83 opening video at 60 HZ in hardware mode has flashes of video visible with default settings. With no deinterlacing it is fully visible but it's flickering. In both cases the emulator keeps losing and acquiring the resolution. 


https://github.com/user-attachments/assets/52529ea5-3eeb-49f9-92e7-92a37251b789

Look at OSD resolution info

**Comment by AmandaRoseChaqueta** [comment]
#13701 seems to fix the flickering issue with the NTSC version, could someone test this on the JP and PAL 60 hz version?

**Comment by AmandaRoseChaqueta** [comment]
I can confirm that the PR fixes the NTSC version AND doesn't break PAL60hz :D
